libs/components/core/package.json

```diff
@@ -19,6 +19,7 @@
     "@angular/cdk": "^17.3.1",
     "@angular/common": "^17.3.1",
     "@angular/core": "^17.3.1",
+    "@angular/forms": "^17.3.1",
```

Comment:
For the new help library we're adding a schematic for the new peer dep. Do you think that would be worth it here? I imaging most all SPAs already have this dep